azure-pipelines.yml

```diff
@@ -0,0 +1,322 @@
+variables:
+- group: sonarsource-build-variables
+- name: BuildParameters.solution
+  value: SonarLint.VisualStudio.Integration.sln
+name: $(Build.BuildId)
+trigger:
+  branches:
+    include:
+    - master
+    - dev/*
+    - feature/*
+    - branch-*
+  batch: True
+jobs:
+- job: Phase_1
+  strategy:
+    maxParallel: 2
+    matrix:
+      2019:
+        vsTargetVersion: 2019
+      2022:
+        vsTargetVersion: 2022      
+  displayName: Phase 1
```

Comment:
Can we have meaningful names for the phases? would make it easier

Reply:
I've kept the original naming. But it makes sense to make it something else.